src/Access/Common/AccessRightsElement.h

```diff
@@ -17,10 +18,8 @@ struct AccessRightsElement
     Strings columns;
     String parameter;
 
-    bool any_database = true;
-    bool any_table = true;
-    bool any_column = true;
-    bool any_parameter = false;
+    bool wildcard = false;
```

Comment:
Is wildcard always related to the last specified field from list `{database, table, columns, parameter}`?

And what about `columns` - this field can contains two or more columns - will `wildcard` work for all those columns then? Can you give an example?

Please write comments here - this structure definitely needs comments now.